storage/csi-cloning.md

```diff
@@ -0,0 +1,124 @@
+---
+title: CSI Cloning
+authors:
+  - "@chuffman"
+reviewers:
+  - "@gnufied”
+  - “@jsafrane”
+approvers:
+  - TBD
+  - "@..."
+creation-date: 2019-09-05
+last-updated: 2019-09-12
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# csi-cloning
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs] 
+
+## Summary
+In the past storage drivers have been provided either in-tree or via external provisioners. With the move to advocating CSI as the way to deliver storage drivers, Red Hat needs to ensure the CSI drivers have comparable functionality to the in-tree drivers.
```

Comment:
There is no cloning support for in-tree drivers, it's available only to CSI.

IMO we should say that we want upstream snapshot feature available in OpenShift.

Reply:
I've updated the Summary based on your suggestion.